mobile/cmp/navigator/impl/swipe/Swiper.js

```diff
@@ -48,10 +48,9 @@ class LocalModel extends HoistModel {
     @action refreshStart()              {this.refreshProgress = 0}
     @action refreshEnd()                {this.refreshProgress = null}
 
-    constructor(navigatorModel) {
-        super();
+    onLinked() {
         makeObservable(this);
```

Comment:
Was definitely on the fence.  I felt avoiding the three lines of boilerplate was worth it.  

This new method -- whatever we call it-- would typically be specified instead of a constructor, and we know it is immediately called right after the constructor, so a safe enough time to do it.  So we are talking about a lot of repetitive boilerplate potentially for little gain.

But agree that this may muddy the waters.

Reply:
took your advice tom.  safer to always do this